### PR TITLE
Fix: Remove project files in case of failed neu create command execution.

### PR DESCRIPTION
**Pull request by Yash-271120**
## Description
Now on unsuccessful neu create due to internet connection problem, the created files get deleted

***
### Fixes(#142)

### CODE REVIEW

**Comment by pathange-s** [comment]
A better description could be: Fix: Remove project files in case of failed `neu create` command execution.

File: src/modules/creator.js [hunk 1]
```diff
@@ -25,6 +25,7 @@ module.exports.createApp = async (binaryName, template) => {
     catch(err) {
         utils.error('Unable to download resources from internet.' +
                     ' Please check your internet connection and template URLs.');
+                    fs.rmdirSync(`../${binaryName}`, { recursive: true });
```
Comment (pathange-s) [inline src/modules/creator.js:28]: Check indentation. should align with `utils.error..`

**Review by pathange-s — CHANGES_REQUESTED** [review]

**Comment by Yash-271120** [comment]
@pathange-s I have made necessary changes please check.

**Comment by pathange-s** [comment]
LGTM.
Btw you could have used `git reset` to undo a commit. Well, let Shalitha approve these changes and workflows run. Then, a merge 🚀

**Review by shalithasuranga — APPROVED** [review]
Looks good.. Thanks :tada:

**Pull request merged**